src/Symfony/Component/HttpFoundation/Session/Storage/Handler/RedisSessionHandler.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Session\Storage\Handler;
+
+/**
+ * Redis based session storage handler based on the Redis class
+ * provided by the PHP redis extension.
+ *
+ * Direct port of MemcachedSessionHandler to Redis.
+ *
+ * @see http://php.net/redis
+ *
+ * @author Drak <drak@zikula.org>
```

Comment:
?

Reply:
The code is/was taken verbatim from his implementation of `MemcachedSessionHandler`, seems appropriate.

As we're moving further away from it with features / changes suggested by @nicolas-grekas, I'll probably remove it.